trunner/long_test.py

```diff
@@ -0,0 +1,140 @@
+#
+# Phoenix-RTOS test runner
+#
+# Long Test Module - Test Runner support for the phoenix-rtos-ports test suites
+#
+# Copyright 2022 Phoenix Systems
+# Authors: Damian Loewnau
+#
+
+from .tools.color import Color
+
+
+LONG_TESTS = ['busybox', 'mbedtls']
+
+
+class LongTestHarnessFactory:
```

Comment:
We do not need this class only for "long tests". We could create factory for "3rd party" harnesses (unity, busybox, embetls) and reduce duplicated code using inheritance